sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Azure.Monitor.OpenTelemetry.LiveMetrics.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="Azure.Core" />
     <PackageReference Include="OpenTelemetry" />
     <PackageReference Include="OpenTelemetry.Exporter.Console" VersionOverride="1.6.0" />
+    <PackageReference Include="System.Diagnostics.PerformanceCounter" VersionOverride="7.0.0" />
```

Comment:
This is needed to read Performance Counters (CPU & Memory) to enable POC. Will work on removing this in the future.

Reply:
We may to keep this one forever, there is no good way to get this value.

Reply:
I'm fine with this dependency, as System.Diagnostic is a core dependency of OTel.
Application Insights is able to read perf counters without an extra dependency. I need more time to investigate how that works. For short term, this is fine. :)

Reply:
AI SDK uses  `System.Diagnostics.PerformanceCounter` - https://github.com/microsoft/ApplicationInsights-dotnet/blob/de66d679ff32f5a74553edbf52b10b9dc57ded70/WEB/Src/PerformanceCollector/PerformanceCollector/Perf.csproj#L32